server/command.go

```diff
@@ -240,6 +249,168 @@ func (p *Plugin) executeShowCommand(args *model.CommandArgs) (*model.CommandResp
 	return &model.CommandResponse{}, nil
 }
 
+func (p *Plugin) executeShowLinksCommand(args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
+	links, err := p.store.GetLinks()
+	if err != nil {
+		return p.cmdError(args.UserId, args.ChannelId, "Something went wrong.")
+	}
+
+	if len(links) == 0 {
+		return p.cmdError(args.UserId, args.ChannelId, "No links present.")
+	}
+
+	p.sendBotEphemeralPost(args.UserId, args.ChannelId, "Please wait while your request is being processed.")
+	go p.SendLinksWithDetails(args.UserId, args.ChannelId, links)
+	return &model.CommandResponse{}, nil
+}
+
+func (p *Plugin) SendLinksWithDetails(userID, channelID string, links []*storemodels.ChannelLink) {
+	var sb strings.Builder
+	sb.WriteString("| Mattermost Team | Mattermost Channel | MS Teams Team | MS Teams Channel | \n| :------|:--------|:-------|:-----------|")
+	gotErrors := false
+	wg := sync.WaitGroup{}
+	batchSize := 10
```

Comment:
Do we need an exported const for this as this batchSize in future this could be used in some other place ?

Reply:
I don't think so.